content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -701,6 +702,10 @@ Each feature gate is designed for enabling/disabling a specific feature:
   risk of collision. See
   [Avoiding collisions](/docs/concepts/services-networking/service/#avoiding-collisions)
   for more details.
+- `SidecarContainers`: Allow `restartPolicy` to be set to `Always` when there are
+  sidecar containers (restartable init containers) in a pod. See
```

Comment:
```suggestion
- `SidecarContainers`: Allow setting the `restartPolicy` of an init container to
  `Always` so that the container becomes a sidecar container (restartable init containers).
  See
```